pyanaconda/modules/storage/bootloader/efi.py

```diff
@@ -204,6 +254,19 @@ def __init__(self):
         self._packages64 = ["grub2-efi-aa64", "shim-aa64"]
 
 
+class Aarch64EFISystemdBoot(EFISYSTEMD):
+    _serial_consoles = ["ttyAMA", "ttyS"]
+    _efi_binary = "\\systemd-bootaa64.efi"
+
+    def __init__(self):
+        super().__init__()
+        # we can't yet add shim-aa64 to list list because it pulls in grub2-efi-aa64, which breaks install-kernel
```

Comment:
that's strange, I wonder why shim would pull grub2...

Reply:
I took the comment out, since I think we are in agreement that shim should be out of the picture, so its redundant as shim won't be installed once comps is updated.